sqldelight-bazel-rules/actions/java/com/squareup/tools/sqldelight/cli/Main.kt

```diff
@@ -0,0 +1,42 @@
+package com.squareup.tools.sqldelight.cli
+
+import com.beust.jcommander.JCommander
+import java.io.BufferedOutputStream
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.StandardOpenOption
+import java.util.zip.ZipEntry
+import java.util.zip.ZipOutputStream
+
+fun main(args: Array<String>) {
```

Comment:
i think what we actually want to commit short term is this function and the dependencies it has, in a new module like `sqldelight-compiler-wrapper`. I'm happy to do that work of pulling it out

then it enables the bazel rules to live elsewhere, maybe in repo but not sure if we want that yet. I don't think having the prerelease stuff makes sense to merge

Reply:
Ideally with [clikt](https://ajalt.github.io/clikt/) if we're going to be maintaining it.

Reply:
As I said, this was ported directly. I plan to migrate off of jcommander but just not in this PR. I was probably going with xenomachina vs. clikt, but I don't have a super strong preference.

And yeah - I definitely think there's a good case for the CLI itself to live elsewhere.

Reply:
I ended up yanking out the prerelease stuff. I had set this PR to draft because I hadn't worked that out yet. In its present state, it has no prerelease, the sample builds, etc.